### PR TITLE
feat(core): updated tooltip ui

### PR DESCRIPTION
**Pull request by vardidzel**

### CODE REVIEW

File: core/src/components/cat-select/cat-select.tsx [hunk 1]
```diff
@@ -149,7 +149,8 @@ export class CatSelect {
   }
 
   @Watch('value')
-  setValueHandler(value?: any) { // eslint-disable-line @typescript-eslint/no-explicit-any
+  setValueHandler(value?: any) {
+    // eslint-disable-line @typescript-eslint/no-explicit-any
```
Comment (geralrodrigues) [inline core/src/components/cat-select/cat-select.tsx:153]: to pass the check of any (also prettier check), you can use eslint-disable-next-line above the method

File: core/src/components/cat-select/cat-select.tsx [hunk 2]
```diff
@@ -149,12 +149,12 @@ export class CatSelect {
   }
 
   @Watch('value')
-  setValueHandler(value?: any) { // eslint-disable-line @typescript-eslint/no-explicit-any
+  setValueHandler(value?: never) {
```
Comment (fynnfeldpausch) [inline core/src/components/cat-select/cat-select.tsx:152]: ehh what? Why is the type `never`

File: core/src/components/cat-select/cat-select.tsx [hunk 3]
```diff
@@ -149,12 +149,12 @@ export class CatSelect {
...
     if (this.resetItemsOnNextValueChange) {
       this.choice?.removeActiveItems(-1);
     }
     this.resetItemsOnNextValueChange = true;
-    this.choice?.setChoiceByValue(value);
+    this.choice?.setChoiceByValue(value || '');
```
Comment (fynnfeldpausch) [inline core/src/components/cat-select/cat-select.tsx:157]: I am also unsure about this change

**Review by fynnfeldpausch — CHANGES_REQUESTED** [review]

**Comment by fynnfeldpausch** [comment]
Prettier fails

**Comment by vardidzel** [comment]
@fynnfeldpausch I have no idea how it was working before, but already fixed.

**Review by fynnfeldpausch — APPROVED** [review]

**Pull request merged**